### ISSUE TITLE
calico-felix should be able to self report its version

### ISSUE THREAD

**Issue by tomdee** [projectcalico/felix#1112]
As I understand it at the moment, the version is mastered in setup.py. I would like to be able to run `calico-felix --version` and have that version reported to me.

**Comment by fasaxc** [comment]
This is already fixed under #1102:
```
$ bin/calico-felix --version
Version:    1.4.1b2-127-g7461598
Build date: 2016-09-27T15:37:27+0000
Git commit: 746159892fe6985e91c3e2985baaffe002d4a1a8
```

**Comment by fasaxc** [comment]
Still mastered in `setup.py` but we now have a `--version` flag under #1122.  @neiljerram is also working on an improved git tag-based release process.
